transcendence-app/Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN npm clean-install && npm run build-packages
 RUN npm run build -w transcendence-app
 VOLUME ${TRANSCENDENCE_APP_DATA}
 ENTRYPOINT ["/usr/bin/dumb-init", "--"]
-CMD ["npm", "run", "start-app-dev"]
+CMD ["sh", "-c", "npm run knex:migrate:dev -w transcendence-app && npm run knex:seed:run -w transcendence-app && npm run start-app-dev"]
```

Comment:
are we sure we want to seed the database for the correction?

Reply:
Good point. I've updated it. Thanks for the suggestion.